verticapy/pipeline/_ingest.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+"""
+Copyright  (c)  2018-2024 Open Text  or  one  of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless  required  by applicable  law or  agreed to in
+    writing, software  distributed  under the  License is
+    distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+    OR CONDITIONS OF ANY KIND, either express or implied.
+    See the  License for the specific  language governing
+    permissions and limitations under the License.
+    """
+"""
+This script runs the Vertica Machine Learning Pipeline Ingestion
+"""
+
+from ._helper import required_keywords, execute_and_return
+from verticapy._utils._sql._sys import _executeSQL
+
+
+def is_valid_delimiter(delimiter: str) -> bool:
```

Comment:
same here, organise per section. And try to put the functions in the right place. Here it looks like a helper function.